std_msgs/Header header
uint8[100] digitalOutput
float32[20] analogOutput